crates/sui-bridge-indexer/src/postgres_manager.rs

```diff
@@ -91,27 +92,52 @@ pub fn read_sui_progress_store(pool: &PgPool) -> anyhow::Result<Option<Transacti
     }
 }
 
-pub fn get_latest_eth_token_transfer(
-    pool: &PgPool,
-    finalized: bool,
-) -> Result<Option<DBTokenTransfer>, Error> {
-    use crate::schema::token_transfer::dsl::*;
+pub fn read_eth_progress_store(pool: &PgPool) -> anyhow::Result<(u64, u64)> {
+    let mut conn = pool.get()?;
+    let val: Option<EthProgressStore> = eth_progress_store::dsl::eth_progress_store
+        .select(EthProgressStore::as_select())
+        .first(&mut conn)
+        .optional()?;
+    match val {
+        Some(val) => Ok((
+            val.earliest_block_synced as u64,
+            val.latest_block_synced as u64,
+        )),
+        None => Ok((0, 0)),
+    }
+}
 
-    let connection = &mut pool.get().unwrap();
+pub fn update_earliest_block_synced(
+    pool: &PgPool,
+    earliest_block: u64,
+) -> Result<(), anyhow::Error> {
+    let mut conn = pool.get()?;
+    diesel::insert_into(schema::eth_progress_store::table)
```

Comment:
can we use diesel::update instead? https://docs.diesel.rs/1.4.x/diesel/fn.update.html

Reply:
Copied the Sui Progress Store func above, but if update makes more sense.